### DISCUSSION TITLE
notify api

### DISCUSSION THREAD

**Discussion by rahulnrt** [WalletConnect/walletconnect-monorepo#3980] [Developer Support] [answered]
Dear team I have published my project on Explorer there is one option notify api I am not able to enable it 

I want to integrate push notifications in our dapp app

**Comment by glitch-txs** [comment]
what's the error?

**Comment by rahulnrt** [comment]
I have no idea. There is one toggle icon to enable it but toggle icon is
disable

**Reply by glitch-txs** [reply]
have you followed this?
https://docs.walletconnect.com/cloud/verify

**Comment by rahulnrt** [comment]
The URL you have shared is a verified API that I already enabled.

The problem is I want to enable notify API because when the customer
transaction is complete I can send the push notification related to this
transaction
https://docs.walletconnect.com/api/notify/about

**Reply by boidushya** [reply]
Hey @rahulnrt , could you check if the dapp url is verified for notify with proper did.json hosted?
Further, could you also provide your projectID?

**Comment by rahulnrt** [comment]
Yes, the did.json file hosted the right dapp URL.
ProjectId:  f84843162a1d79bfd4a4bd8a1d350923

**Reply by boidushya** [reply]
Have you tried checking for CORs error in your console, after clicking the notify api toggle?

**Comment by rahulnrt** [comment]
Yes, I did. So what is solution

**Reply by boidushya** [reply, marked as answer]
Seems like you need to enable CORS on your domain. You can test your CORS configuration through [this website](test-cors.org) . Then, depending on your framework you need to set a CORS policy.

**Comment by rahulnrt** [comment]
Thanks, I'll check it out.

**Comment by rahulnrt** [comment]
what about the projectId that I have sent will you delete

**Reply by boidushya** [reply]
You can delete it if you want, not an issue
FYI in case youre using Next.js
[This might be helpful for CORS](https://github.com/WalletConnect/gm-hackers/blob/4e38ea016dd7c44e5e5d2f7c5d26bfbb59b81b75/next.config.js#L8)

**Comment by rahulnrt** [comment]
Do you have any access?
